server/sonar-web/src/main/js/apps/quality-profiles/components/ProfileActions.js

```diff
@@ -152,6 +182,30 @@ export default class ProfileActions extends React.PureComponent {
               {translate('delete')}
             </a>
           </li>}
+
+        {this.state.copyFormOpen &&
+          <CopyProfileForm
+            onClose={this.closeCopyForm}
+            onCopy={this.handleProfileCopy}
+            onRequestFail={this.props.onRequestFail}
+            profile={profile}
+          />}
+
+        {this.state.deleteFormOpen &&
+          <DeleteProfileForm
+            onClose={this.closeDeleteForm}
+            onDelete={this.handleProfileDelete}
+            onRequestFail={this.props.onRequestFail}
+            profile={profile}
+          />}
+
+        {this.state.renameFormOpen &&
+          <RenameProfileForm
+            onClose={this.closeDeleteForm}
```

Comment:
Copy-paste error probably, currently the rename form can't close when clicking on cancel